bitnami/node-exporter/README.md

```diff
@@ -130,6 +130,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `updateStrategy.type`                         | The update strategy type to apply to the DaemonSet                                                                                                                 | `RollingUpdate`         |
 | `updateStrategy.rollingUpdate.maxUnavailable` | Maximum number of pods that may be made unavailable                                                                                                                | `1`                     |
 | `hostNetwork`                                 | Expose the service to the host network                                                                                                                             | `true`                  |
+| `hostPID`                                     | Allows visibility of processes on the host, potentially leaking information such as environment variables and configuration                                        | `false`                 |
```

Comment:
As it was previously set to true, can you set the default value to true?

Reply:
Hello, hostPID was changed to default - value is true. @Mauraza